<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocBookViewer::PrintDialog</class>
 <widget class="QDialog" name="DocBookViewer::PrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Items to print</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="itemsChooser">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Pages options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Page size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="paperSize">
        <item>
         <property name="text">
          <string>A4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pagesCountLabel">
        <property name="text">
         <string>Pages count: unknown</string>
        </property>
        <property name="buddy">
         <cstring>estimatePagesButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="estimatePagesButton">
        <property name="text">
         <string>Estimate pages count</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DocBookViewer::PrintDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DocBookViewer::PrintDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
